tests/ubersmith_request_get_test.py

```diff
@@ -66,6 +65,26 @@ def test_api_get_method_returns_with_arguments(self, request_mock):
 
         expected_call()
 
+    @patch('ubersmith_client.ubersmith_request_get.requests')
+    def test_api_get_support_ticket_submit_allow_file_upload(self, request_mock):
+        expected_files = {'attach[0]': ('filename.pdf', b'filecontent')}
+        expected_call = self.expect_a_ubersmith_call_with_files(requests_mock=request_mock,
+                                                                method='support.ticket_submit',
+                                                                subject='that I use to know',
+                                                                body='some body',
+                                                                returning=a_response_data(data='42'),
+                                                                files=expected_files)
+
+        ubersmith_api = ubersmith_client.api.init(self.url, self.username, self.password, use_http_get=True)
+
+        response = ubersmith_api.support.ticket_submit(subject='that I use to know',
+                                                       body='some body',
+                                                       files=expected_files)
```

Comment:
The ubersmith documentation says that ticket_submit takes "attach[x]" as a parameter 

https://ibb.co/cPohXb

So wouldn't it make more sense that this should be called like
```
ubersmith_api.support.ticket_submit(subject='that I use to know', body='some body', **{"attach[0]": ('filename.pdf', b'filecontent')})
```

Because we usually respect ubersmith's contract

Reply:
about that, ubersmith does not respect it's contract and ignore the parameters "attach[x]", it does only look info $_FILES so it would be more like
```
ubersmith_api.support.ticket_submit(subject='that I use to know', body='some body', **{"files": {"attach[0]": ('filename.pdf', b'filecontent')}})
```

that why I've used it that way in the test

Reply:
lôl.... ok